File: unity/Assets/Hallway/Constructor/RPG/firstVisitTest.js
#pragma strict

var Bootstrap : GameObject;

function OnTriggerEnter() {
	var scr1 : RPGParser = Bootstrap.GetComponent.<RPGParser>();
	if (!scr1.RPG.firstVisitTest) {
		scr1.Achievement("Первое посещение комнаты тестирования!\n+10 очков!", 10);
		scr1.RPG.firstVisitTest = true;
	}
}

File: unity/Assets/Hallway/Constructor/RPG/firstVisitTest.cs
using UnityEngine;

public class firstVisitTest : MonoBehaviour
{
    public GameObject Bootstrap;
    void OnTriggerEnter()
    {
        RPGParser rpgParser = Bootstrap.GetComponent<RPGParser>();
        if (!rpgParser.RPG.firstVisitTest)
        {
            rpgParser.Achievement("Первое посещение комнаты тестирования!\n+10 очков!", 10);
            rpgParser.RPG.firstVisitTest = true;
        }
    }
}
